db/rdbms-schema/src/main/resources/db/changelog/rdbms-exporter/changesets/8.9.0.xml

```diff
@@ -786,4 +789,31 @@
     </modifySql>
   </changeSet>
 
+  <changeSet id="create_cluster_variable_table" author="camunda">
+    <createTable tableName="${prefix}CLUSTER_VARIABLE">
+      <column name="CLUSTER_VARIABLE_ID" type="VARCHAR(${userCharColumnSize})">
+        <constraints primaryKey="true"/>
+      </column>
+      <column name="VAR_NAME" type="VARCHAR(${userCharColumnSize})"/>
+      <column name="TYPE" type="VARCHAR(255)"/>
+      <column name="DOUBLE_VALUE" type="DOUBLE"/>
+      <column name="LONG_VALUE" type="BIGINT"/>
+      <column name="VAR_VALUE" type="VARCHAR(8192)"/>
+      <column name="VAR_FULL_VALUE" type="CLOB"/>
+      <column name="IS_PREVIEW" type="BOOLEAN"/>
+      <column name="RESOURCE_ID" type="VARCHAR(${userCharColumnSize})"/>
+      <column name="SCOPE" type="VARCHAR(${userCharColumnSize})"/>
+    </createTable>
+
+    <createIndex tableName="${prefix}CLUSTER_VARIABLE"
+      indexName="${prefix}IDX_CLUSTER_VARIABLE_RESOURCE_ID">
+      <column name="RESOURCE_ID"/>
+    </createIndex>
```

Comment:
❓ Should it be a composite index? I guess the access pattern would be mostly by scope and resource id, and not just by resource id.